sdk/appcenter/src/main/java/com/microsoft/appcenter/http/HttpClientNetworkStateHandler.java

```diff
@@ -67,82 +64,34 @@ public void reopen() {
 
     @Override
     public synchronized void onNetworkStateUpdated(boolean connected) {
-        if (connected) {
+        if (connected && mCalls.size() > 0) {
             AppCenterLog.debug(LOG_TAG, "Network is available. " + mCalls.size() + " pending call(s) to submit now.");
-        } else {
-            AppCenterLog.debug(LOG_TAG, "Network is down. Pausing " + mCalls.size() + " network call(s).");
```

Comment:
We still need a log that says network is down.

Reply:
It's logged in `NetworkStateHelper`